Dockerfile.rhel7

```diff
@@ -0,0 +1,14 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 AS builder
```

Comment:
we still can do the build on rhel-8 and copy build result to rhel7 image I'm assuming?